src/domain/fileUploader.repository.interface.ts

```diff
@@ -0,0 +1,21 @@
+import type { Note } from './entities/Note';
```

Comment:
I'm not sure that file uploader should depend on Note

Reply:
for now, i created separate repository with note attachments